src/Components/Header/style.jsx

```diff
@@ -0,0 +1,205 @@
+import styled from 'styled-components';
+import { Home } from '@styled-icons/entypo/Home';
+import { User } from '@styled-icons/boxicons-solid/User';
+import { Bars } from '@styled-icons/fa-solid/Bars';
+
+export const SHeader = styled.div`
```

Comment:
You must remove all these exports and put your const into the last export without the default word